src/Process/3dTilesProcessing.js

```diff
@@ -265,9 +234,6 @@ export function init3dTilesLayer(view, scheduler, layer, rootTile) {
             tile.updateMatrixWorld();
             layer.tileset.tiles[tile.tileId].loaded = true;
             layer.root = tile;
-            // The extent is calculated but it's never used in 3D tiles process
-            layer.extent = boundingVolumeToExtent(layer.crs || view.referenceCrs,
-                tile.boundingVolume, tile.matrixWorld);
```

Comment:
Do you have an idea why the extent was calculated in the first place ?

Reply:
no idea

Reply:
@gchoqueux maybe you know some history behind this ?